app/assets/v2/css/onboarding_v2.css

```diff
@@ -0,0 +1,114 @@
+.benefits {
+  margin: 16px 0 24px;
+}
+
+.benefits p>* {
+  display: block;
+  margin: auto;
+  text-align: center;
+}
+
+.benefits>div:nth-child(2n) {
+  border-left: 1px solid #ccc;
+}
+
+.benefits>div {
+  border-bottom: 1px solid #ccc;
+  padding: 16px 6px 8px;
+}
+
+.benefits>div:nth-child(5),
+.benefits>div:nth-child(6) {
+  border-bottom: 0;
+}
+
+.benefits .fa {
+  width: 48px;
+}
+
+.testimonial {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0 16px;
+  max-height: 92px;
+  align-items: center;
+}
+
+.testimonial>*>* {
+  display: block;
+}
+.modal {
+  overflow: visible;
+}
+.modal-content {
+  margin-top: 256px;
+}
```

Comment:
This will overwrite already existing classes, if you need to overwrite custom elements you need to add custom classes. 
Also modal overflow visible is needed here? if what you want is scroll you can read about it on getboostrap.com as exist already classes for it.

Reply:
Fixed